jscomp/test/bs_unwrap_test.ml

```diff
@@ -0,0 +1,49 @@
+external log1 :
+  (
+    [ `Pair of string * int
+    | `Int of int
```

Comment:
In that case there is a test for both below. 

I don't have a strong preference between "unwrap" and "unbox" as the term we use here.